docs/CI-CD.md

```diff
@@ -0,0 +1,93 @@
+and they currently include:
+
+* [Publish](https://github.com/lf-edge/eve/actions?query=workflow%3APublish) publishes various EVE build artifacts into an OCI Registry
+* [Pulls](https://github.com/lf-edge/eve/actions?query=workflow%3APRs) manages pull requests
+* [Apache Yetus](https://github.com/lf-edge/eve/actions?query=workflow%3A%22Apache+Yetus%22) runs static, lint-like [Apache Yetus](http://yetus.apache.org/) checks on EVE
+* [Eden](https://github.com/lf-edge/eve/actions?query=workflow%3AEden) runs integration tests from EVE's sister project [Eden](https://github.com/lf-edge/eden)
+
+Since EVE community generally frowns on branch-heavy development most of the descriptions below
+will only apply to the master branch (unless stated otherwise).
+
+## Publish
+
+This is the only action that uses both x86 and ARM64 runners. It gets triggered on either updates
+to the `master` branch or creation of release tags. Publish produces a set of tagged artifacts and
+pushes them into EVE's OCI Registry containers. Logic responsible for figuring out correct tags
+for these artifacts resides in EVE's top-level Makefile and can be summarized as:
+
+* If Publish gets triggered on a release tag (release tags are the ones with `x.y.z` names) that
+  compares as grater than all the other release tags currently present in a repo that release tag
+  is considered to be the latest. All artifacts originating from the Publish triggered on the latest
+  tag get a matching `x.y.z` OCI registry tags AND get an additional symbolic tag `latest`. Since
+  the artifacts are typically retrieved with a `docker pull lfedge/eve-XXX` this policy corresponds
+  well with the expectation that such a pull is expected to retrieve `lfedge/eve-XXX:latest` even
+  though it is a mere alias for `lfedge/eve-XXX:x.y.z`. Another crucial distinction of a Publish
+  action triggered on the latest tag is that it gets the benefit of a full (from scratch) build
+  as opposed to using pre-cached artifacts. This is done to ensure that EVE builds can be rebuilt
+  in the future in a self-contained manner but it does, of course, increase the build time dramatically.
+* If Publish gets triggered on a release tag that is NOT considered to be the `latest` (for example when
+  building .z patch releases for older x.y.(z-1) releases) the OCI registry artifacts only get a matching
+  `x.y.z` tag and no additional aliases are generated. The build is also typically much faster since it
+  uses pre-cached artifacts in OCI registry if the corresponding source package in the EVE tree didn't
+  change. Note, however, that building a patch release for the previous latest tag becomes a new latest
+  tag and thus subject to the latest tag treatment (e.g. `x.y.z` being latest means `x.y.(z+1)` will now
+  compare as greater than any other tag since it is greater than `x.y.z` itself).
+* If Publish gets triggered on a `master` branch all EVE packages in an OCI registry get two tags assigned:
```

Comment:
I assume this means on a push to `master` branch, i.e. merged PR? Might help to clarify that.

Reply:
done